src/Enums/ElementIds.ts

```diff
@@ -0,0 +1,6 @@
+export enum ElementIds {
```

Comment:
Firstly, we don't need to create a separate folder for enums. 
Secondly, enums can be in a constant file so there is no need to create a separate file.